test/onnx/test_onnx_export.py

```diff
@@ -60,3 +61,35 @@ def break_is_registered_op_api(opname, domain, version):
                               operator_export_type=OperatorExportTypes.ONNX_ATEN_FALLBACK)
         onnx_model = onnx.load_from_string(f.getvalue())
         self.assertAtenOp(onnx_model, "clamp", "Tensor")
+
+
+    def _helper_test_to_(self, cast_fn: Callable[[torch.Tensor], torch.Tensor]):
+        """Run a generic test with the specified cast_fn
+
+        `cast_fn` will be converted into a `torch.jit.script` function
+        and it is meant to wrap a `aten::to` during export to prevent
+        hard-coded devices
+        """
+        cast_fn = torch.jit.script_if_tracing(cast_fn)
```

Comment:
Add comment explaining why script_if_tracing is needed

Reply:
I did: "wrap a `aten::to` during export to prevent hard-coded devices"